openedx/features/marketplace/tests/view_tests.py

```diff
@@ -19,14 +21,38 @@ def setUpClass(cls):
         theme = SiteTheme(site=site, theme_dir_name='philu')
         theme.save()
 
+    def setUp(self):
+        super(MarketplaceViewTests, self).setUp()
+        self.user = UserFactory()
```

Comment:
As you are here, 
I know this isn't in changes but It would be great if you simplify setUpClass method by removing philu site configuration code and use `configure_philu_theme` method instead.